test/MusicStore.Test/ManageControllerTest.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
got it. The other files in this MusicStore have no license header, and I'll add the license header for the new files, but let me update the existing files in a seperate PR.